### PR TITLE
Update byacc

### PR DESCRIPTION
**Pull request by iarspider**
* Remove all references to github (abandoned?)
* Add proper version

### CODE REVIEW

**Comment by iarspider** [comment]
Pinging @darmac who wrote the original recipe

**Comment by darmac** [comment]
I got someone to concern about this issue.
Is there anything to Discussing?
@iarspider

**Comment by iarspider** [comment]
I don't expect anything to change. Also, maybe you want to become a maintainer of this package?

**Comment by darmac** [comment]
> I don't expect anything to change. Also, maybe you want to become a maintainer of this package?

Hmmm, thanks for you inviting, but so sorry, I'm not working for this anymore.
If anything is wrong, I can still find someone to try fix that.

File: var/spack/repos/builtin/packages/byacc/package.py [hunk 1]
```diff
@@ -13,10 +13,11 @@ class Byacc(AutotoolsPackage):
     documentation. Specifications that take advantage of undocumented
     features of AT&T Yacc will probably be rejected."""
 
-    homepage = "https://github.com/grandseiken/byacc"
-    git      = "https://github.com/grandseiken/byacc.git"
+    homepage = "https://invisible-island.net/byacc/byacc.html"
+    url      = "https://invisible-island.net/datafiles/release/byacc-20210808.tar.gz"
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/byacc/package.py]: Does this url exist?

**Review by alalazo — CHANGES_REQUESTED** [review]
```console
$ wget https://invisible-island.net/datafiles/release/byacc-20210808.tar.gz
--2021-09-13 09:55:02--  https://invisible-island.net/datafiles/release/byacc-20210808.tar.gz
Resolving invisible-island.net (invisible-island.net)... 192.124.249.12
Connecting to invisible-island.net (invisible-island.net)|192.124.249.12|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2021-09-13 09:55:03 ERROR 404: Not Found.
```

File: var/spack/repos/builtin/packages/byacc/package.py (hunk 1 above)
Comment (iarspider) [reply var/spack/repos/builtin/packages/byacc/package.py]: Looks like it no longer works. Changed to FTP

**Comment by bernhardkaindl** [comment]
@iarspider I wanted to propose up update the sha256sum of FTP url of 20210808 but could not find it at first glance
I hope the additional commit which updates it works for you too. ;-)

**Comment by iarspider** [comment]
> 
> 
> @iarspider I wanted to propose up update the sha256sum of FTP url of 20210808 but could not find it at first glance
> I hope the additional commit which updates it works for you too. ;-)

Sure, thanks!

**Review by bernhardkaindl — APPROVED** [review]
Built it and all three dependencies of it. LGTM.

**Comment by bernhardkaindl** [comment]
@alalazo It is ready to be merged, testing is completed by me by building all dependents.
Please approve, thanks.

**Review by alalazo — APPROVED** [review]

**Pull request merged**